<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchList</class>
 <widget class="QWidget" name="SwitchList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_left">
     <property name="styleSheet">
      <string notr="true">width: 58px;
height: 58px;
background: #22A4B3;
border-radius: 29px;
font-size: 12px;
font-weight: bold;
color: #000000;

image: url(:/suzhou/btn_arrow_left.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_msg">
     <property name="toolTipDuration">
      <number>-4</number>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 402px;
height: 58px;
background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_right">
     <property name="styleSheet">
      <string notr="true">width: 58px;
height: 58px;
background: #22A4B3;
border-radius: 29px;
font-size: 12px;
font-weight: bold;
color: #000000;

image: url(:/suzhou/btn_arrow_right.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
